content/hi/docs/contribute/review/for-approvers.md

```diff
@@ -0,0 +1,202 @@
+
+- `/assign` Prow कमांड का उपयोग आवश्यकतानुसार एक विशिष्ट समीक्षक को PR सौंपने के लिए करें। यह अतिरिक्त महत्वपूर्ण है जब कोड योगदानकर्ताओं से तकनीकी समीक्षा का अनुरोध करने की बात आती है।
+
+  {{< note >}}
+  यह देखने के लिए कि कौन की तकनीकी समीक्षा  प्रदान कर सकता है , Markdown फ़ाइल के शीर्ष पर  `reviewers` फ़ील्ड देखें।
+  {{< /note >}}
+
+- यह सुनिश्चित करें कि PR [विषय](/docs/contribute/style/content-guide/) और [शैली](/docs/contribute/style/style-guide/) गाइड का अनुसरण करते हैं; यदि ऐसा नहीं होता है तो लेखक को गाइड के प्रासंगिक भाग से लिंक करें।
+- यदि लागू हो, तो GitHub **Request Changes** विकल्प का उपयोग करके PR लेखक को परिवर्तनों का सुझाव दें।
+- यदि आपके सुझावों पर अमल किया जाता हैं, तो GitHub पर समीक्षा स्थिति बदलने के लिए
+`/approve` या `/lgtm` Prow कमांड का उपयोग करें।
+
+## किसी अन्य व्यक्ति के पीआर में शामिल हों
+
+PR टिप्पणियाँ छोड़ना मददगार होता है, लेकिन कई बार ऐसा भी हो सकता है जब आपको इसके बजाय किसी अन्य व्यक्ति के PR में प्रतिबद्ध होने की आवश्यकता हो।
+
+जब तक वे स्पष्ट रूप से नहीं पूछें, तब तक किसी अन्य व्यक्ति के लिए "अधिकार" न लें आप, या आप एक लंबे समय से परित्यक्त PR को फिर से जीवित करना चाहते हैं। जबकि यह तेज हो सकता है अल्पावधि में, यह व्यक्ति को योगदान करने के अवसर से वंचित करता है।
+
+आपके द्वारा उपयोग की जाने वाली प्रक्रिया इस बात पर निर्भर करती है कि आपको पहले से मौजूद फ़ाइल को संपादित करने की आवश्यकता है या नहीं पीआर के दायरे में, या एक फाइल जिसे PR ने अभी तक छुआ नहीं है।
+
+
+यदि निम्न में से कोई एक चीज़ सच है तो आप किसी और के PR में शामिल नहीं हो सकते :
+
+- यदि PR लेखक ने अपनी शाखा को सीधे
+  [https://github.com/kubernetes/website/](https://github.com/kubernetes/website/) रिपॉजिटरी में धकेल दिया। केवल पुश एक्सेस वाला समीक्षक ही किसी अन्य उपयोगकर्ता के PR के लिए प्रतिबद्ध हो सकते हैं।
+
+  {{< note >}}
+  लेखक को प्रोत्साहित करें कि अगली बार PR खोलना खोलने से पहले वह अपनी शाखा को पहले 
+  अपने फोर्क की ओर धकेले।
+  {{< /note >}}
+
+- पीआर लेखक स्पष्ट रूप से अनुमोदकों से संपादन की अनुमति नहीं देता है।
+
+## समीक्षा के लिए Prow कमांड
+
+[Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md) एक
+कुबेरनेट्स-आधारित CI/CD सिस्टम है जो पुल अनुरोधों (PRs) पर विभिन्न कार्यों को चलाता है।
+Prow चैटबॉट-शैली कमांड को सक्षम करता है हमें संपूर्ण कुबेरनेट्स संगठन में GitHub क्रियाओं को
+संभालने के लिए, जैसे [लेबल जोड़ना और हटाना](#adding-and-removing-issue-labels), मुद्दों को बंद करना, और एक अनुमोदक नियुक्त करना। `/<command-name>` प्रारूप का उपयोग करके GitHub टिप्पणियों के रूप में Prow कमांड दर्ज करें।
+
+समीक्षकों और अनुमोदकों द्वारा उपयोग किए जाने वाले सबसे आम Prow कमांड हैं:
+
+{{< table caption="समीक्षा के लिए Prow कमांड" >}}
+Prow कमांड | भूमिका प्रतिबंध | विवरण
+:------------|:------------------|:-----------
+`/lgtm` | संगठन के सदस्य | संकेत करता है कि आपने PR की समीक्षा पूरी कर ली है और परिवर्तनों से संतुष्ट हैं।
+`/approve` | अनुमोदक | विलय के लिए एक PR को मंजूरी देता हैं।
```

Comment:
I understand that merge = विलय. However, merging changes via a PR is a technical term and it might confuse people. WDYT?
```suggestion
`/approve` | अनुमोदक | Merge के लिए एक PR को मंजूरी देता हैं।
```